src/measurements.jl

```diff
@@ -895,3 +895,77 @@ function _fiducial_WH(::Type{T}, d::Integer) where {T}
 end
 
 _fiducial_WH(d::Integer) = _fiducial_WH(ComplexF64, d)
+
+"""
+    state_discrimination_min_error(
+        ρ::Vector{<:AbstractMatrix{T}},
+        q::Vector{<:AbstractFloat} = fill(1/length(ρ), length(ρ));
```

Comment:
```suggestion
        q::Vector{<:AbstractFloat} = fill(real(T)(1)/length(ρ), length(ρ));
```
Otherwise `q` is always a `Float64` and your precision goes down the drain.